### PR TITLE
[NEP-12806] Reminder Dashboard - Add Tag to View

### PR DESCRIPTION
**Pull request by footnoise**
Ticket: https://jobready.atlassian.net/browse/NEP-12806

Add tags to dashboard items

### CODE REVIEW

File: src/components/ListWidgetItem.js [hunk 1]
```diff
@@ -1,12 +1,18 @@
 var React = require('react');
 
 var ListWidgetItem = React.createClass({
+    renderTags: function() {
+        return (this.props.tags || []).map(function(name) {
+            return(<span key={Math.random()} class="badge badge-secondary">{name}</span>)
```
Comment (krazedkrish) [inline src/components/ListWidgetItem.js:6]: @footnoise , I am curious on why you put `key={Math.random()}`

Comment (footnoise) [reply src/components/ListWidgetItem.js:6]: @krazedkrish `key` should be unique number, if we using like `{idx}`(just index of element in array) in that case we can get warning that similar object with the the key is presented in DOM.
`Math.random()` is pretty safe it generates you a good unique numbers.

**Review by krazedkrish — APPROVED** [review]
LGTM 👍

**Review by krazedkrish — APPROVED** [review]
LG

**Pull request merged**